README.md

```diff
@@ -87,11 +121,66 @@ This API should be integrated in your shopping cart and there are two simple ste
   _`-H 'cache-control: no-cache' `_
   _`-H 'Accept: image/png' `_
 
-- **Notes:**
+### Placing a new order
 
-  For any questions please send an email to hello@storydots.app
+- **URL**
 
+  _/order_
 
-## **Example code snippets:**
+- **Method:**
+
+  `POST` - assigns a StoryDots code to your new purchase
+
+- **Request Headers**
+
+  **x-api-key:** your private API key that authorizes the request and creates a new order linked to a specific shop
+  **Content-Type:** application/json
+  
+- **Body params**
+
+The params should be sent as a JSON, see the sample call below. These are the params the endpoint will expect:
+  **buyerEmail**: the email address of the buyer to which we will send en email with the recording link
+  **buyerName**: the name of the buyer, helps us customize the experience
+  **orderId**: your internal ID of the order, we will store this ID in case we need to troubleshoot any issues
```

Comment:
store this ID to allow you to trace the order if needed

Reply:
done